### PR TITLE
DAOS-7509 documentation: Update DAOS HW documentation

### PR DESCRIPTION
**Pull request by cdurf1**
Skip-build: true
Skip-test: true

Signed-off-by: craig <craig.durfey@intel.com>

### CODE REVIEW

File: docs/admin/hardware.md [hunk 1]
```diff
@@ -1,113 +1,136 @@
...
+depends on (e.g., ISA-L[^1], SPDK[^2], PMDK[^3], and DPDK[^4]) can take
+advantage of Intel SSE[^5] and AVX[^6] extensions.
 
-Some success was also reported by the community on running the DAOS client
-on 64-bit ARM processors configured in Little Endian mode. That being said,
-ARM testing is not part of the current DAOS CI pipeline and is thus not
-validated on a regular basis.
+The DAOS client is regularly tested on 64-bit ARM processors configured
+in Little Endian mode. The same build instructions that are used for
+x86\_64 are applicable for ARM builds as well. DAOS and its dependencies
```
Comment (Michael-Hennecke) [inline docs/admin/hardware.md:39]: the DAOS client is not regularly tested on ARM64 ... please keep the original text.

File: docs/admin/hardware.md [hunk 2]
```diff
@@ -1,113 +1,136 @@
-# Hardware Requirements
+Hardware Requirements
+=====================
```
Comment (Michael-Hennecke) [inline docs/admin/hardware.md:2]: please keep original MD heading markup using #, ##, etc.

File: docs/admin/hardware.md [hunk 3]
```diff
@@ -1,113 +1,136 @@
...
+------------------
 
-As illustrated in the figure below, a DAOS storage system is deployed
-as a **Pooled Storage Model**. The DAOS servers can run on dedicated
-storage nodes in separate racks. This is a traditional pool model
-where storage is uniformly accessed by all compute nodes. In order
-to minimize the number of I/O racks and to optimize floor space,
-this approach usually requires high-density storage servers.
+As illustrated in the figure below, a DAOS storage system can be
+deployed in two different ways:
```
Comment (Michael-Hennecke) [inline docs/admin/hardware.md:11]: figure dropped, so delete the reference to it?

File: docs/admin/hardware.md [hunk 4]
```diff
@@ -1,113 +1,136 @@
...
-The DAOS network layer relies on libfabrics and supports OFI providers
-for Ethernet/sockets, InfiniBand/verbs and RoCE.
-An RDMA-capable fabric is preferred for better performance. DAOS can support
-multiple network interfaces by binding different engines and different client
-processes to individual network cards.
+The DAOS data plane relies on OFI libfabrics[^7] and supports OFI
+providers for Ethernet/sockets and InfiniBand/verbs. An RDMA-capable
+fabric is preferred for better performance. DAOS can support multiple
+rails by binding different instances of the DAOS server to individual
+network cards.
```
Comment (Michael-Hennecke) [inline docs/admin/hardware.md:50]: please keep the original text - "rails" have a special meaning that usually does not apply here, and it *is* the engine and not the server that is meant here.

File: docs/admin/hardware.md [hunk 5]
```diff
@@ -1,113 +1,136 @@
...
+depending on the read/write balance of the application workload. Since
+NVMe SSDs have read faster than they write, a 200Gbps PCIe4 x4 NIC can
+be balanced for read only workloads by 4 NVMe4 x4 SSDs, but for write
+workloads by 8 NVMe4 x4 SSDs. The capacity of the SSDs will determine
+the minimum capacity of the Optane PMem DIMMs needed to provide the 6%
+ratio for DAOS metadata.
 
-![](./media/Fig_074.png)
+![](media/image2.png){width="5.138888888888889in"
+height="3.1944444444444446in"}
```
Comment (Michael-Hennecke) [inline docs/admin/hardware.md:104]: remove image resizing tags?

File: docs/admin/hardware.md [hunk 6]
```diff
@@ -1,113 +1,136 @@
...
-responsible for detecting the storage and network affinity and
-starting the I/O Engines accordingly.
+per CPU, then accessing only the persistent memory and PCI devices local
+to that CPU from that server instance. The DAOS control plane is
+responsible for detecting the storage and network affinity and starting
+the I/O Engines accordingly.
 
-![](./media/Fig_075.png)
+![](media/image3.png){width="5.513888888888889in"
+height="5.791666666666667in"}
```
Comment (Michael-Hennecke) [inline docs/admin/hardware.md:130]: remove image resizing tags?

File: docs/admin/hardware.md [hunk 7]
```diff
@@ -118,9 +141,28 @@ DAOS assumes that fault domains are hierarchical and do not overlap. For
 instance, the first level of a fault domain could be the racks and the
 second one, the storage nodes.
 
-For efficient placement and optimal data resilience, more fault
-domains are better. As a result, it is preferable to distribute storage
-nodes across as many racks as possible.
+For efficient placement and optimal data resilience, more fault domains
+are better. As a result, it is preferable to distribute storage nodes
+across as many racks as possible.
+
```
Comment (Michael-Hennecke) [inline docs/admin/hardware.md:147]: use in-place markdown HREFs [text](link) instead of footnotes?

**Review by Michael-Hennecke — CHANGES_REQUESTED** [review]
see inline comments...

**Comment by kprantis** [comment]
Any update on addressing the above review feedback?

**Comment by cdurf1** [comment]
@kprantis  - I'll have it in by EOD

File: docs/admin/hardware.md (hunk 4 above)
Comment (Michael-Hennecke) [reply docs/admin/hardware.md:40]: still referring to "rails".... which isn't how most DAOS systems are deployed: most DAOS systems have their multiple network interfaces connected to the same "rail", not to multiple "rails". so the original text that simply says "multiple network interfaces" is the more general and more appropriate description.

**Review by Michael-Hennecke — APPROVED** [review]

**Review by johannlombardi — APPROVED** [review]

**Comment by johannlombardi** [comment]
Please back port to release/2.0.

**Pull request merged**